src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -371,10 +371,7 @@ protected Colony(final int id, @Nullable final Level world)
         colonyStateMachine.addTransition(new TickingTransition<>(INACTIVE, () -> true, this::updateState, UPDATE_STATE_INTERVAL));
         colonyStateMachine.addTransition(new TickingTransition<>(UNLOADED, () -> true, this::updateState, UPDATE_STATE_INTERVAL));
         colonyStateMachine.addTransition(new TickingTransition<>(ACTIVE, () -> true, this::updateState, UPDATE_STATE_INTERVAL));
-        colonyStateMachine.addTransition(new TickingTransition<>(ACTIVE, () -> true, () -> {
```

Comment:
@someaddons any reason this wasnt a normal one before?

Reply:
this was a normal one xD just a lamda since it was a void